tests/test_validator.py

```diff
@@ -1159,7 +1159,7 @@ def test_verify_unique_flows_duplicate_names(
 
 
 # testing for None and empty string
-@pytest.mark.parametrize("way_of_emptiness", ["", '""'])
+@pytest.mark.parametrize("way_of_emptiness", ['""', "''"])
```

Comment:
Changed because the first example now raises a `SchemaError` not a validation error.

Reply:
Ok then we can remove the parametrization. I think they are now equivalent